gluon-mesh-layer3-common: Allow lifetime configuration in site (#3570)
freifunk-gluon/gluon: docs/package/gluon-radvd.rst

gluon-radvd
===========

This package provides service files and configuration endpoints for the IPv6
router advertisement daemon.

Arguments
---------

This package provides skeleton service files for ``uradvd``.
It requires another package to provide the arguments ``uradvd`` is supposed to
be launched with.

> :doc:`gluon-mesh-batman-adv` and ``gluon-mesh-layer3-common`` are two packages
> providing such arguments.
> 
> This allows to have ``uradvd`` announce default routes for Layer 3 meshes,
> while only announcing prefixes for Layer 2 meshes.

Reviewer: ```suggestion
:doc:`gluon-mesh-batman-adv` and ``gluon-mesh-layer3-common`` are two packages
providing such arguments in order to announce default routes for Layer 3 meshes,
while only announcing prefixes for Layer 2 meshes.
```

> This allows to have ``uradvd`` announce default routes for layer three meshes,
> while only announcing prefixes for layer two meshes.

Reviewer 2: ```suggestion
This allows to have ``uradvd`` announce default routes for layer 3 meshes,
while only announcing prefixes for layer 2 meshes.
```

I think writing the number down is the more typical way and what we've used in the past for layer 2 or layer 3. But no strong opionion.

Author: I think it would be correct to refer to the layer with the number three, and spell it out as it's a small number, or use the OSI naming and refer to it by its title.
In that case we can use the digit, but would capitalize it as name `OSI Layer 3` or `Layer 3`.

Looking at out code base I think this is personal preference at best and we should spend less time on such things in general.

Author: resolved in 0805f8015859703ae94a9334f67eb796d2d879b8.
Now matches the VXLAN section in the VPN docs.

site.conf
---------

radvd.preferred_lifetime : optional
    - the span of time during which the address can be freely used as a source

>       and destination for traffic. Should be less or equal valid-lifetime.

Reviewer 2: ```suggestion
      and destination for traffic. Should be less or equal valid_lifetime.
```

Probably makes sense to use the exact spelling of the other option.

Author: The texts currently match their description in `uradvd`.
If this is a strong opinion, we can remove the dash in both docs,
but I'd just leave it as it is.

Reviewer 2: (I would argue that gluon-radvd is an abstraction layer and therefore should reference itself where possible. But this really isn't something i want to spend time on arguing. So fine with me. :) )
